test/integration/default/inspec/default_spec.rb

```diff
@@ -0,0 +1,8 @@
+describe command('curl http://localhost:4000/directory') do
```

Comment:
Can you please change this to using the http resource in InSpec? And the same for below.